### PR TITLE
[Enhancement] Add beneficiary properties

### LINKED ISSUE #173: [Enhancement] Some missing enrollment fields
- [ ] Beneficiaries
  - [ ] Beneficiary percentage
  - [ ] Beneficiary address
  - [ ] Beneficiary SSN
  - [ ] Add additional beneficiaries

### PR DESCRIPTION
**Pull request by petersonbrock**
closes #173 

Screen Shot 2020-11-25 at 1 29 12 PM
Screen Shot 2020-11-25 at 1 30 32 PM

### CODE REVIEW

File: app/controllers/account/benefits/enrollment/beneficiaries.js [hunk 1]
```diff
@@ -1,27 +1,82 @@
...
+  async addBenef (type) {
+    let beneficiary = this[`${type}Life`],
+        { addressLine1, addressLine2, addressCity, addressState, addressZip } = await this.auth.user,
+        address;
 
-    if (this.primaryLife.dirtyType === 'created' && this.primaryLife.name) {
-      this.model.beneficiaries.addObject(this.primaryLife);
+    if (beneficiary.address && this[`${type}UniqueAddress`]) {
+      address = beneficiary.address;
+    } else {
```
Comment (JavaLavaMT) [inline app/controllers/account/benefits/enrollment/beneficiaries.js:46]: indention seems off

File: app/controllers/account/benefits/enrollment/beneficiaries.js [hunk 2]
```diff
@@ -1,27 +1,82 @@
...
   }
 
   @action
   afterSave () {
-    this.model.beneficiaries.forEach(b => {
-      if (!b.id) {
-        b.destroyRecord();
-      }
-    });
+    this.model.beneficiaries.filterBy('id', null).invoke('destroyRecord');
```
Comment (JavaLavaMT) [inline app/controllers/account/benefits/enrollment/beneficiaries.js:75]: Nice add 🔥

File: app/templates/account/benefits/enrollment/beneficiaries.hbs [hunk 3]
```diff
@@ -2,51 +2,147 @@
...
+    </div>
+  {{else}}
+    <h4 class="ui center aligned header">No contingent life beneficiaries</h4>
+  {{/if}}
+
+  {{#if this.contingentLifeAdding}}
+    <div class="ui divider"></div>
+    <div class="two fields">
+      <div class="field">
+        <label for="prime-life-name">Name</label>
```
Comment (JavaLavaMT) [inline app/templates/account/benefits/enrollment/beneficiaries.hbs:135]:
```suggestion
        <label for="contingent-life-name">Name</label>
```

Comment (petersonbrock) [reply app/templates/account/benefits/enrollment/beneficiaries.hbs:135]: good catch

**Review by James1x0 — APPROVED** [review]

**Pull request merged**